taxonomies/food/categories.txt

```diff
@@ -4297,7 +4297,7 @@ origins:en: en:France, fr:Bourgogne
 fr:Eau-de-vie de vin de la Marne
 de:Branntwein aus Marne, Branntwein aus dem Marne Tal
 nl:Wijn-eau-de-vies uit de Marne
-origins:en: en:France, fr:Marne
```

Comment:
fr:Marne not existing is only a warning, so I don't know if we should remove those.

@stephanegigandet and @teolemon what do you think ?

Reply:
With @teolemon we think we should not remove this, but maybe, in another PR, add those to origins.txt

Reply:
Agreed, it would be best to add them to origins.txt